### DISCUSSION TITLE
Location to upload my pdf files.

### DISCUSSION THREAD

**Discussion by shrutig2000** [freedmand/semantra-python#73] [Q&A]
I have cloned Semantra in my local system, but unable to locate where to upload my pdf for semantic search.
I am running the application using svelte method.
![semantra](https://github.com/freedmand/semantra/assets/143679602/35b9bbf7-4594-47e0-9031-b85a69bb1bec)

**Comment by underwater** [comment]
try here: docs/example_docs (the included book is here: docs/example_docs/hamlet.pdf)
